app/src/main/java/com/aptoide/walletpermissionssample/MainActivity.java

```diff
@@ -9,26 +9,38 @@
 import android.support.annotation.NonNull;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.widget.AppCompatEditText;
+import android.util.Log;
 import android.view.View;
 import android.widget.Button;
 
+import org.w3c.dom.Text;
+
+import java.util.Set;
+
 public class MainActivity extends AppCompatActivity {
 
     private Button btnPermissions;
+    private AppCompatEditText walletAddressText;
```

Comment:
Since in xml you're using EditText, you should also use it here instead of AppCompatEditText

Reply:
fixed on latest commit